Lecture3/Task1/Voucher.java

```diff
@@ -0,0 +1,41 @@
+abstract class Voucher {
+    private String name;
+    private double price;
+    private String country;
+    private String startDate;
+    private int countDays;
+    private int discount;
+    private double cost;
+
+    int getCountDays() {
+        return countDays;
+    }
+
+    double getCost() {
+        return cost;
+    }
+
+    public abstract String getFood();
+
+    public abstract String getTransfer();
+
+    public void initVoucher(int i) {
```

Comment:
initVoucher() method sets the required attributes hence actually it should be a constructor.

Reply:
fixed